### PR TITLE
add faq

### PR DESCRIPTION
**Pull request by mirnawong1**
adding faq to help users solve  403 service token error [per slack thread](https://dbt-labs.slack.com/archives/C04H4G1NET0/p1700587758226669).

### CODE REVIEW

File: website/docs/faqs/Troubleshooting/ip-restrictions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            "status": {
+                "code": 403,
+                "is_success": False,
+                "user_message": ("Forbidden: Access denied"),
+                "developer_message": None,
+            },
+            "data": {
+                "account_id": <account_id>,
+                "user_id": <user_id>,
+                "is_service_token": <boolean describing if it's a service token>,
```
Comment (nm0s) [inline website/docs/faqs/Troubleshooting/ip-restrictions.md]: nit: but maybe update to `<boolean describing if it's a service token request>` for more clarity

**Review by nm0s — APPROVED** [review]
left one non-blocking comment. Thanks for making these changes :)

File: website/docs/faqs/Troubleshooting/ip-restrictions.md (hunk 1 above)
Comment (mirnawong1) [inline website/docs/faqs/Troubleshooting/ip-restrictions.md]:
```suggestion
                "is_service_token": <boolean describing if it's a service token request>,
```

**Comment by mirnawong1** [comment]
thank you so much @nm0s and folded in your feedback ✨

**Pull request merged**